## Lab 5: Randomization and Experiments

Let's continue to use the anchor data for today's coding practice. Students are tasked to guess the maximum speed of a house cat in miles per hour. 

In the "high" condition, students were prompted with 

"Is the maximum speed of an average house cat FASTER or SLOWER than **40** miles per hour?"

In the "low" condition, students were prompted with

"Is the maximum speed of an average house cat FASTER or SLOWER than **3** miles per hour?"

The student then had to provide an estimate for the speed of a house cat. We want to learn: does anchoring (receiving messages) have an effect on students' estimates? Let's assume that students were randomly assigned to each group.

-------

**Data Dictionary/Codebook**

`cat`: estimated speed in mph

`before`: 1 = student knew the exercise beforehand, 0 = student did not know

`live.cats`: 1 = student lives with cats, 0 = student does not live with cats

`cond`: 1 = student was exposed to the "high" condition, 0 = student was exposed to the "low" condition 

In [1]:
# Load data
anchor <- read.csv("./Data/anchoring_data_FA20.csv")
head(anchor)

cat before live.cats cond
1  2  0      0         0   
2  1  0      0         1   
3 30  0      0         1   
4 25  0      0         1   
5 17  0      0         0   
6 50  0      0         1

Let's then "clean" the dataset. We want to remove some potentially problematic variables; we do NOT want to study people who (1) know about the goal of the experiment or (2) have lived with a cat, since they'll be more knowledgeable and have different answers.

**Question 1**

We do so by subsetting the data, so we only get (1) the participant has NOT lived with a cat, and (2) the participant was NOT aware of the experiment/exercise. We save this to a new variable `focus_anchor`.

In [2]:
# Could you try one-liner codes to generate the new variable focus_anchor?

focus_anchor <- subset(anchor, before==0 & live.cats==0)
head(focus_anchor)

cat before live.cats cond
1  2  0      0         0   
2  1  0      0         1   
3 30  0      0         1   
4 25  0      0         1   
5 17  0      0         0   
6 50  0      0         1

**Question 2**

What is the treatment effect of people assigned to the "high group" compared to those assigned to the "low" group?


In [3]:
# estimatr is an R package providing a range of commonly-used linear estimators
library(estimatr) 

In [4]:
# What codes are needed in the brackets? You may refer to the R cheatsheet on bcourses
difference_in_means(cat ~ cond, focus_anchor, condition1 = '0', condition2 = '1') 

Design:  Standard 
     Estimate Std. Error  t value     Pr(>|t|) CI Lower CI Upper       DF
cond  17.4881    2.45516 7.122995 5.837135e-10 12.59595 22.38025 73.87644

**Question 3**

How should we interpret the estimate?

Students primed with the high condition message will estimate the cat speed 17.4881 miles per hour higher than the estimation from students primed with the low condition message.

**Question 4**

What is the standard error?

2.757204